tests/test_app.py

```diff
@@ -0,0 +1,119 @@
+import os
+from unittest.mock import patch
+
+import pytest
+from dotenv import load_dotenv, find_dotenv
+
+import app
+
+
+def _mock_response(
+        status=200,
+        content="CONTENT",
+        json_data=None,
+        raise_for_status=None):
+    class MockResponse:
+        def __init__(self, json_data, status_code):
+            self.json_data = json_data
+            self.status_code = status_code
+
+        def json(self):
+            return self.json_data
+
+        def raise_for_status(self):
+            return True
+
+    return MockResponse(json_data=json_data, status_code=status)
+
+
+todos_json = [
+    {
+        "id": "5f57f133fe6fec1ff7e25d63",
+        "dateLastActivity": "2020-09-21T17:39:25.603Z",
+        "name": "Valid Item",
+        "due": None
+    },
+    {
+        "id": "5f57f133fe6fec1ff7e25d64",
+        "dateLastActivity": "2020-09-21T17:39:25.603Z",
+        "name": "Some other Item",
+        "due": None
+    },
+    {
+        "id": "5f57f133fe6fec1ff7e25d65",
+        "dateLastActivity": "2020-09-21T17:39:25.603Z",
+        "name": "An Item with a due",
+        "due": "2020-09-22T10:00:00.603Z"
+    }
+]
+
+board_lists_json = [
+    {
+        "id": "5f4dfb5dc9f1a151051ce7a6",
+        "name": "Todo"
+    },
+    {
+        "id": "5f4dfb63e12bc1039a2bca84",
+        "name": "Done"
+    }
+]
+
+
+def tearDown(self):
+    try:
+        board_id = os.getenv("TODO_BOARD_ID")
+        Tre
+    except OSError as oserr:
+        print(oserr)
+
+
+@pytest.fixture
+def client():
+    mock_get_request = patch('trello_request.requests.request')
```

Comment:
No harm in patching the specific use, but it would be reasonable here to just patch `requests.request` generally, and would be a bit more tolerant to filenames changing elsewhere in your application